File: src/Entity/Reclamation.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;

/**
 * Reclamation
 *
 * @ORM\Table(name="reclamation")
 *@ORM\Entity(repositoryClass="App\Repository\ReclamationRepository")
 */
class Reclamation
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="type", type="string", length=255, nullable=false)
     */
    private $type;

    /**
     * @var string
     *
     * @ORM\Column(name="ido", type="string", length=11, nullable=false)
     */
    private $ido;

    /**
     * @var string
     *
     * @ORM\Column(name="sujet", type="string", length=255, nullable=false)
     *@Assert\Length(min=4,max=35)
     */
    private $sujet;

    /**
     * @var string
     *
     * @ORM\Column(name="description", type="string", length=255, nullable=false)
     *@Assert\Length(min=4,max=35)
     */
    private $description;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date", type="datetime", nullable=false, options={"default"="current_timestamp()"})
     */
    private $date = 'current_timestamp()';

    /**
     * @var string
     *
     * @ORM\Column(name="etat", type="string", length=255, nullable=false, options={"default"="'en attente '"})
     */
    private $etat = '\'en attente \'';

    /**
     * @var string|null
     *
     * @ORM\Column(name="idU", type="string", length=11, nullable=true, options={"default"="NULL"})
     */
    private $idu = 'NULL';

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getType(): ?string
    {
        return $this->type;
    }

    public function setType(string $type): self
    {
        $this->type = $type;

        return $this;
    }

    public function getIdo(): ?string
    {
        return $this->ido;
    }

    public function setIdo(string $ido): self
    {
        $this->ido = $ido;

        return $this;
    }

    public function getSujet(): ?string
    {
        return $this->sujet;
    }

    public function setSujet(string $sujet): self
    {
        $this->sujet = $sujet;

        return $this;
    }

    public function getDescription(): ?string
    {
        return $this->description;
    }

    public function setDescription(string $description): self
    {
        $this->description = $description;

        return $this;
    }

    public function getDate(): ?\DateTimeInterface
    {
        return $this->date;
    }

    public function setDate(\DateTimeInterface $date): self
    {
        $this->date = $date;

        return $this;
    }

    public function getEtat(): ?string
    {
        return $this->etat;
    }

    public function setEtat(string $etat): self
    {
        $this->etat = $etat;

        return $this;
    }

    public function getIdu(): ?string
    {
        return $this->idu;
    }

    public function setIdu(?string $idu): self
    {
        $this->idu = $idu;

        return $this;
    }


}

File: src/com/mycompany/myap/entities/Reclamation.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.myap.entities;

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

public class Reclamation {
       
       public Reclamation(int id,String type, String sujet, String description, String date, String etat) {
        this.type = type;
        this.id = id;
        this.sujet = sujet;
        this.description = description;
        this.date = date;
        this.etat = etat;
    }    
        
    public Reclamation(String type,String ido, String sujet, String description, String idu) {
        this.type = type;
        this.ido = ido;
        this.sujet = sujet;
        this.description = description;
        this.idu = idu;
    }

   
    public Reclamation(String description, int id) {
        this.id = id;
        this.description = description;
    }
    public Reclamation(int id, String etat) {
        this.id = id;
        this.etat = etat;
    }

    
    public int id;
    private String type;
    private String ido;
    private String sujet;
    private String description;
    private String date;
    private String etat;
    private String idu;

    public Reclamation(int id, String type, String ido, String sujet, String description, String date, String etat, String idu  ) {
        this.id = id;
        this.type = type;
        this.ido = ido;
        this.sujet = sujet;
        this.description = description;
        this.date = date;
        this.etat = etat;
        this.idu = idu;
    }

    public Reclamation(int id, String type, String ido, String sujet, String description, String date, String etat) {
        this.id = id;
        this.type = type;
        this.ido = ido;
        this.sujet = sujet;
        this.description = description;
        this.date = date;
        this.etat = etat;
    } 
  

    public Reclamation(String type, String sujet, String description) {
        this.type = type;
        this.sujet = sujet;
        this.description = description;
    }
    public Reclamation() {
    }
    public Reclamation(String user) {
        this.idu = idu ;
    }

    public Reclamation(int id, String type, String sujet, String description) {
        this.id = id;

        this.type = type;
        this.sujet = sujet;
        this.description = description;
    }

    public Reclamation(String type, String ido, String sujet, String description, String etat, String idu) {
        this.type = type;
        this.ido = ido;
        this.sujet = sujet;
        this.description = description;
        this.etat = etat;
        this.idu = idu;
    }

   

    @Override
    public String toString() {
        return "Reclamation{" + "id=" + id + ", type=" + type + ", ido=" + ido + ", sujet=" + sujet + ",description=" + description + ", date=" + date + ", etat=" + etat + ", User=" + idu + '}';
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getIdo() {
        return ido;
    }

    public void setIdo(String ido) {
        this.ido = ido;
    }

    public String getSujet() {
        return sujet;
    }

    public void setSujet(String sujet) {
        this.sujet = sujet;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public String getEtat() {
        return etat;
    }

    public void setEtat(String etat) {
        this.etat = etat;
    }

    public String getUser() {
        return idu;
    }

    public void setUser(String user) {
        this.idu = idu;
    }

}
